src/tests/unit/__vitest_data__/instant-loan.ts

```diff
@@ -22,7 +22,7 @@ const branchHoldings: HoldingsV3[] = [
         periodical: undefined,
         materialGroup: {
           name: "standard",
-          description: "I am supposed to be matched, but I am not available"
+          description: "I NOT supposed to be matched"
```

Comment:
Are you sure that the text should be changed here?
A lot of work went into writing the tests.

Note: "I NOT supposed to be matched" is not grammatically correct.

Reply:
I'm adding a new commit to clarify why it should fail